src/components/Navbar/links.tsx

```diff
@@ -10,6 +10,7 @@ export const navLinks: NavLink[] = [
   { title: "Events", slug: "events" },
   { title: "Visit Us", slug: "visit" },
   { title: "Contact Us", slug: "contact" },
+  {title: "Profile", slug: "profile"},
```

Comment:
What do title and slug mean here? If I add `alt: false`, will there be any changes?

Reply:
title here is the name of the link on the home page, in this case "Profile". Then it will transfer me to a new page on the website and the last part of the url will be the slug so in this case "profile" in http://localhost:3000/profile is named by the slug. After testing, adding an alt:false to line 13 does not do anything.